### DISCUSSION TITLE
Can't hit 'enter' in PiVPN installer

### DISCUSSION THREAD

**Discussion by rollinthunderr** [pivpn/pivpn#1843] [Q&A]
I'm trying to install PiVPN on my Raspberry Pi, but once the automated installer opens, I'm not able to move forward. As I understand it, I should be able to select "Ok" by hitting 'Enter,' but that doesn't do anything. I've tried it on both PuTTY and Terminal. Is there a simple solution?

**Comment by orazioedoardo** [comment]
Can you try connecting the raspbperry pi to a monitor and keyboard and try from there?

**Reply by ctivanovich** [reply]
@orazioedoardo  In either case, you need a terminal, and the installer can't move forward in the terminal, whether via SSH or via Desktop GUI. 

Edit: I was able to move forward with install by downloading the repo and running auto_install.sh directly.

**Reply by ElkyBoy** [reply]
You don't need Enter, use Tab and Space instead.
